components/security-mgt/org.wso2.carbon.security.mgt.ui/src/main/resources/web/keystoremgt/delete-keystore-finish-ajaxprocessor.jsp

```diff
@@ -0,0 +1,67 @@
+~
+~ WSO2 Inc. licenses this file to you under the Apache License,
+~ Version 2.0 (the "License"); you may not use this file except
+~ in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~    http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing,
+~ software distributed under the License is distributed on an
+~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+~ KIND, either express or implied.  See the License for the
+~ specific language governing permissions and limitations
+~ under the License.
+-->
+<%@page import="org.apache.axis2.context.ConfigurationContext" %>
+<%@page import="org.wso2.carbon.CarbonConstants" %>
+<%@page import="org.wso2.carbon.security.ui.SecurityUIConstants" %>
+<%@page import="org.wso2.carbon.security.ui.client.KeyStoreAdminClient" %>
+<%@ page import="org.wso2.carbon.ui.CarbonUIMessage" %>
+<%@page import="org.wso2.carbon.ui.CarbonUIUtil" %>
+<%@page import="org.wso2.carbon.utils.ServerConstants" %>
+<%@ page import="java.text.MessageFormat" %>
+<%@ page import="java.util.ResourceBundle" %>
+<%@ page import="org.owasp.encoder.Encode" %>
+
+<%
+    String httpMethod = request.getMethod();
+    if (!"post".equalsIgnoreCase(httpMethod)) {
+        response.sendError(HttpServletResponse.SC_METHOD_NOT_ALLOWED);
+        return;
+    }
+
+    String forwardTo = null;
+    String keyStore = request.getParameter("keyStore");
+    String BUNDLE = "org.wso2.carbon.security.ui.i18n.Resources";
+    ResourceBundle resourceBundle = ResourceBundle.getBundle(BUNDLE, request.getLocale());
+    try {
+        String cookie = (String) session.getAttribute(ServerConstants.ADMIN_SERVICE_COOKIE);
+        String backendServerURL = CarbonUIUtil.getServerURL(config.getServletContext(), session);
+        ConfigurationContext configContext =
+                (ConfigurationContext) config.getServletContext().getAttribute(CarbonConstants.CONFIGURATION_CONTEXT);
+        KeyStoreAdminClient client = new KeyStoreAdminClient(cookie, backendServerURL, configContext);
+        client.deleteStore(keyStore);
+        String message = resourceBundle.getString("keystore.delete");
+        forwardTo = "keystore-mgt.jsp";
+        CarbonUIMessage.sendCarbonUIMessage(message, CarbonUIMessage.INFO, request);
+        session.setAttribute(SecurityUIConstants.RE_FETCH_KEYSTORES, Boolean.TRUE);
+    } catch (Exception e) {
+        String message = MessageFormat.format(resourceBundle.getString("keystore.cannot.delete"),
+                new Object[]{e.getMessage()});
+        forwardTo = "keystore-mgt.jsp";
+        CarbonUIMessage.sendCarbonUIMessage(message, CarbonUIMessage.ERROR, request);
+    }
+%>
+
+<script type="text/javascript">
+    function forward() {
+        location.href = "<%=Encode.forJavaScriptBlock(forwardTo)%>";
+    }
+</script>
+
+<script type="text/javascript">
+    forward();
+</script>
```

Comment:
Add a new line for EOL

Reply:
Resolved every instance with 0988fbf923202bead47acc0234ab6d1d24029c1d